test/should_fail/opaque_fail.erl

```diff
@@ -1,7 +1,24 @@
 -module(opaque_fail).
 
--export([use_external/2]).
+-export([use_external/2, update_without_opaque/0, add_to_opaque/0, return_opaque/0]).
 
 -spec use_external(user_types:my_opaque(), integer() | undefined) -> integer().
 use_external(I, undefined) -> I;
 use_external(_, I) -> I.
+
+-spec update_without_opaque() -> ok.
+update_without_opaque() ->
+    _Val = user_types:update_opaque(3),
+    ok.
+
+-spec add_to_opaque() -> ok.
+add_to_opaque() ->
+    Val = user_types:new_opaque(),
+    Val + 1,
+    ok.
+
+-spec return_opaque() -> user_types:my_opaque().
+return_opaque() -> 3.
+
+-spec abc(A) -> A.
+abc(X) -> X.
```

Comment:
Unused function?

Reply:
Good catch, this is probably some leftover from some of my testings. Removed in 71395366.